templates/KnockoutSpa/ClientApp/boot.ts

```diff
@@ -1,8 +1,8 @@
 import './css/site.css';
 import * as ko from 'knockout';
-import { createHistory } from 'history';
 import './webpack-component-loader';
 import AppRootComponent from './components/app-root/app-root';
+var createHistory = require('history').createBrowserHistory
```

Comment:
Is this change deliberate? Could you clarify why it would be needed (hopefully it isn't), and if it is, change the `var` to `const` and add the semicolon at the end of the line?